tests/test_validation_utils.py

```diff
@@ -1,6 +1,73 @@
+import logging
+
+import numpy as np
 import pytest
+import rioxarray  # noqa: F401
+import xarray as xr
+
+from src.pipelines import seas5_pipeline
+from src.utils.validation_utils import validate_dataset, validate_metadata_leadtime
+
+LOGGER = logging.getLogger(__name__)
+
+BASE_ATTRS = [
```

Comment:
Do you think there's any way to reference these from the source code? https://github.com/OCHA-DAP/ds-raster-pipelines/blob/ba68cd0e37c5c55882f0d639b74feb828fa21fd1/src/pipelines/pipeline.py#L74

Would that be best practice? Just don't want to have to change things in multiple places if we adjust these variables.